deps/exokit-bindings/videocontext/src/Video.cpp

```diff
@@ -1,5 +1,6 @@
 #include <Video.h>
 #include "VideoMode.h"
+#include "VideoCamera.h"
```

Comment:
Can we clean these up re angle and non-angle? I guess it was already inconsistent but this PR adds additional inconsistency.